content/docs/tour/basics-deploying.md

```diff
@@ -1,32 +1,3 @@
 ---
-title: Deploying code
-menu:
-  tour:
-    parent: tour
-    weight: 3
+redirect_to: /docs/reference/cli/pulumi_up/
```

Comment:
I found that we didn't have great conceptual documentation on "deploying" so linked to the command itself, which does provide a good overview. Applied to a few other pages as well (e.g. previewing and destroying).